estate/models/estate_property.py

```diff
@@ -0,0 +1,98 @@
+from odoo import fields, models, api
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+
+class Property(models.Model):
+    _name = 'estate.property'
+    _description = 'Real Estate Property'
+    _order = 'id desc'
+
+    # Custom fields
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(string="Available From", copy=False, default=fields.Date.add(fields.Date.today(), months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer(string="Living Area (sqm)")
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection([('n', 'North'), ('e', 'East'), ('s', 'South'), ('w', 'West')])
+
+    # Computed fields
+    total_area = fields.Integer(string='Total Area (sqm)', compute='_compute_total_area')
+    best_price = fields.Float(string='Best Offer', compute='_compute_best_price')
+
+    @api.depends('living_area', 'garden_area')
+    def _compute_total_area(self):
+        for record in self:
```

Comment:
Again more of a preference but I would usually choose a name close to the record I am looping through, so here maybe
```suggestion
        for property in self:
```
So t is clear for the reader what model I am talking about